### ISSUE TITLE
Show Dowgo Ownership of stocks

### ISSUE THREAD

**Issue by antoine-dowgo** [DOWGO/dowgo-dapp-token#7]

**Comment by antoine-dowgo** [comment]
@Tris92 Let me know what you think and what you have in mind

**Comment by Tris92** [comment]
I want to display each stock % owned by dowgo alpha fund not only on the main page (which will show limited information such as company's name and % allocated) but also on a page that we would name "Dowgo Funds", in which much more detailed information from the fund's chosen stocks would be displayed. Additionally, are there any metrics we could retrieve that would give significant information about the green aspect of our chosen stocks? In the end we could really emphasize on the "why" we chose those funds and "how" do we illustrate each company's sustainibility initiatives/results through accurate metrics/announcements/performance/ etc.

**Comment by antoine-dowgo** [comment]
Sounds good! Also with @oscar-dowgo we discussed adding links to bloomberg website showing/proving ownership of these stocks.
How were you planning on retrieving that information? I can create an API endpoint on AWS with that if you want.

**Comment by Tris92** [comment]
Yes I think it would be pretty good to create an API endpoint. What type of data will it generate though? Can I insert them within a table or will it need different formatting?

**Comment by oscar-dowgo** [comment]
For each Token, we have to display companies name, weight/company (%), company sector, company description. 
Then, possibility to reach the API to display other features afterwards. 

I let you chose a format @Tris92 
Here is an example form a ON CHAIN Asset Manager : https://app.enzyme.finance/vault/0xe0f1a74b6f340d1dfefe4b0268f04b23cc665f27/portfolio

**Comment by antoine-dowgo** [comment]
@oscar-dowgo 

>company sector, company description

Where would we get that information? From another API or our own manually filled database (or manually written in the code for the 10 stocks of the Alpha)?

@Tris92 we can discuss the format of the data

**Comment by oscar-dowgo** [comment]
@antoine-dowgo we have to create our data base as I have a classic Bloomberg account. The API account for Bloomberg is far to expensive for the moment. 

Here is the data base : https://docs.google.com/spreadsheets/d/1EMugCUg6-vaDWLZANInsHkVQdCYKeKdi?rtpof=true&authuser=oscar%40dowgo.com&usp=drive_fs

We can create our database with Bloomberg informations for each companies we plan to invest in. I will put the informations on this excel.

**Comment by antoine-dowgo** [comment]
Ok sounds good

**Comment by antoine-dowgo** [comment]
Api ready: https://github.com/DOWGO/dowgo-scripts/issues/27
